docs/source/alfacase/api_reference.rst

```diff
@@ -2,3 +2,214 @@ Full API Reference
 ==================
 
 This page contains the full reference to |sdk| API.
+
+
+Case
+----
+
+.. autoclass:: alfasim_sdk.CaseDescription()
+
+Options
+-------
+
+.. autoclass:: alfasim_sdk.PhysicsDescription()
+
+.. autoclass:: alfasim_sdk.TimeOptionsDescription()
+
+.. autoclass:: alfasim_sdk.NumericalOptionsDescription()
+
+
+Outputs
+-------
+
+.. autoclass:: alfasim_sdk.CaseOutputDescription()
```

Comment:
Because I want to hide the class signature, which is confusing in my opinion.

![image](https://user-images.githubusercontent.com/5083518/100753915-2b21a380-33c9-11eb-9fbd-87c58a355ef5.png)